src/main/groovy/life/qbic/datamodel/dtos/business/CostEstimate.groovy

```diff
@@ -97,11 +98,6 @@ class CostEstimate {
             return this
         }
 
-        Builder selectedCustomerAffiliation(Affiliation selectedCustomerAffiliation) {
-            this.selectedCustomerAffiliation = selectedCustomerAffiliation
-            return this
-        }
-
```

Comment:
wait can we not add the selected customer affiliation to the object now?

Reply:
No it was a relict of the DTO in which only the `modificationDate` was required. 
I adapted the DTO so the `customerAffiliation` has to be provided during the DTO generation and is not optional.